src/internal/operators/combineAll.ts

```diff
@@ -6,6 +6,50 @@ export function combineAll<T>(): OperatorFunction<ObservableInput<T>, T[]>;
 export function combineAll<T>(): OperatorFunction<any, T[]>;
 export function combineAll<T, R>(project: (...values: T[]) => R): OperatorFunction<ObservableInput<T>, R>;
 export function combineAll<R>(project: (...values: Array<any>) => R): OperatorFunction<any, R>;
+/**
+ * Flattens an Observable-of-Observables by applying {@link combineLatest} when the Observable-of-Observables completes.
+ *
+ * ![](combineAll.png)
+ *
+ * `combineAll` takes an Observable of Observables, and collects all Observables from it. Once the outer Observable completes,
+ * it subscribes to all collected Observables and combines their values using the {@link combineLatest}</a> strategy, such that:
+ *
+ * * Every time an inner Observable emits, the output Observable emits
```

Comment:
Hey @JWO719, this is for the markdown list. It would be bold if it ended with another `*`, but it only uses the first `*` and then indented `*` for a second level list.

Reply:
Yeah of course. :D